code/game/machinery/bots/mulebot.dm

```diff
@@ -512,6 +512,16 @@ var/global/mulebot_count = 0
 
 	lock_atom(C, /datum/locking_category/mulebot)
 
+/obj/machinery/bot/mulebot/proc/can_load(var/atom/movable/C)
+	if (!istype(C,/obj/item) && !istype(C,/obj/machinery) && !istype(C,/obj/structure) && !ismob(C))
+		return FALSE
+	if (!emagged)
+		if (istype(C,/obj/machinery/door))
+			return check_access(botcard)
+		if (istype(C, /obj/structure/grille) || istype(C, /obj/structure/window))
```

Comment:
Coincidentally I was working on the exact same PR
I'd rather this be locked behind things being anchored or non-dense/opaque, any similar checks to that.